MyLibrary/Library/Startup.cs

```diff
@@ -0,0 +1,51 @@
+﻿// <copyright file="Startup.cs" company="Peretiatko Anastasiia">
+// Copyright (c) Peretiatko Anastasiia. All rights reserved.
+// </copyright>
+
+using BusinessLogic.DataProvider;
+using BusinessLogic.LibraryService;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Library
+{
+    /// <summary>
+    /// Class for configuration
+    /// </summary>
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddSingleton<ILibraryService, LibraryService>();
+            services.AddSingleton<IDataProvider, ListsDataProvider>();
```

Comment:
Why singleton, not transient or scoped?

Reply:
I checked the work of Transient or Scoped.
They don`t save the state of our Service between requests, and so we get initial list of data with each new request.